### ISSUE TITLE
Question: Redact dot notation paths

### ISSUE THREAD

**Issue by eauc** [davidmarkclements/fast-redact#20]
Hello, I want to redact mongo queries in my logs, eg

```
query: {
   'my.secret.token': 'cndqoncwqon',
}
```

I tried paths like `paths: ["query['my.secret.token']"]` but the validator complains 

```
undefined:9
       if (o.query && o.query['my && o.query['my.secret && o.query['my.secret.token'] != null) {
                                              ^^
 
 SyntaxError: Unexpected identifier
```

any ideas ?
thanks in advance

**Comment by davidmarkclements** [comment]
this is a bug -  I'm looking into :+1:

**Comment by davidmarkclements** [comment]
this is now fixed in v2.0.0

**Comment by eauc** [comment]
thank you very much for taking the time :bowing_man: 

I updated my deps and indeed everything works fine with v2.0.0 :+1:
